Lib/test/test_fractions.py

```diff
@@ -302,6 +302,12 @@ def testFromDecimal(self):
             ValueError, "cannot convert NaN to integer ratio",
             F.from_decimal, Decimal("snan"))
 
+    def test_as_integer_ratio(self):
```

Comment:
I did a comparison of these tests to the existing ones, and noticed that `test_decimal.py` contains coverage for attempting to convert inf (`inf = 1e1000`), -inf, nan (`nan = inf - inf`), and an invalid string to an integer ratio.

```python
    def test_as_integer_ratio(self):
        Decimal = self.decimal.Decimal

        # exceptional cases
        self.assertRaises(OverflowError,
                          Decimal.as_integer_ratio, Decimal('inf'))
        self.assertRaises(OverflowError,
                          Decimal.as_integer_ratio, Decimal('-inf'))
        self.assertRaises(ValueError,
                          Decimal.as_integer_ratio, Decimal('-nan'))
        self.assertRaises(ValueError,
                          Decimal.as_integer_ratio, Decimal('snan123'))
```
For the sake of consistency, should we include that here as well? I'm not certain that it's necessary, but I figured it was worth mentioning.

Reply:
No need.  Fractions don't have that concept.